internal/packages/packagecontent/topackage.go

```diff
@@ -4,16 +4,86 @@ import (
 	"bytes"
 	"context"
 	"path/filepath"
+	"regexp"
+	"sort"
 	"strings"
 
+	manifestsv1alpha1 "package-operator.run/apis/manifests/v1alpha1"
+
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/yaml"
 
 	"package-operator.run/internal/packages"
 )
 
-func PackageFromFiles(ctx context.Context, scheme *runtime.Scheme, files Files) (pkg *Package, err error) {
+var componentFileRE = regexp.MustCompile("^([a-z][a-z0-9-]{0,62})/(.+)$")
```

Comment:
```suggestion
var componentFileRE = regexp.MustCompile(`^([a-z][a-z0-9-]{0,62})/(.+)$`)
```
also use ``a`` for the quotation, not `"a"`, using the latter will break when backspaces are added.

Reply:
done 👍 (in the updated regex)